app/policies/feed_policy.rb

```diff
@@ -12,14 +12,14 @@ def create?
   end
 
   def new_apple?
-    create?
+    update?
   end
 
   def update?
-    PodcastPolicy.new(token, resource.podcast).update?
+    PodcastPolicy.new(token, resource.podcast).update? && !resource.edit_locked?
   end
 
   def destroy?
-    resource.custom? && update? && !resource.apple?
+    resource.custom? && update?
```

Comment:
now allowing deletes - we can edit_lock feeds we do. not want touched, but this way self service folks can not only disable, but delete the integration with Apple